LATEST

```diff
@@ -1,5 +1,5 @@
 {
-  "daml": "2.2.0-snapshot.20220412.9721.0.b7c4bdab",
-  "canton": "2.1.1",
-  "prefix": "2.1.1"
+  "daml": "2.3.0-snapshot.20220505.9861.0.5d8ebd7a",
```

Comment:
The fact that 2.3 is used is because we already bumped the prefix on the Daml repo, right?